src/tyr/route_serializer_osrm.cc

```diff
@@ -1482,7 +1486,12 @@ json::ArrayPtr voice_instructions(const valhalla::DirectionsLeg::Maneuver* prev_
                                   const valhalla::DirectionsLeg::Maneuver& maneuver,
                                   const double distance,
                                   const uint32_t maneuver_index,
-                                  valhalla::odin::EnhancedTripLeg* etp) {
+                                  valhalla::odin::EnhancedTripLeg* etp,
+                                  const valhalla::Options& options) {
+  // narrative builder for custom pre alert instructions
+  MarkupFormatter nullFormatter;
+  std::unique_ptr<NarrativeBuilder> narrative_builder = NarrativeBuilderFactory::Create(options, etp, nullFormatter);
```

Comment:
Also not 100% happy with this. The functions which we don't need the markup formatter, so I'm just passing a null formatter.